src/Containers/Lobby/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Col, Row } from 'antd';
+
+import { Box, CenteredContainer, CenteredDiv, Tags } from 'Components/CommonComponents';
+import Button from 'Components/Button';
+import { PlayerList, StartButton } from './styles';
+import { useState } from 'react';
```

Comment:
Again and again, separate library imports from project imports
```suggestion
import { Col, Row } from 'antd';
import { useState } from 'react';

import { Box, CenteredContainer, CenteredDiv, Tags } from 'Components/CommonComponents';
import Button from 'Components/Button';
import { PlayerList, StartButton } from './styles';
```